src/promptflow/promptflow/_sdk/entities/_flow.py

```diff
@@ -190,11 +196,24 @@ def load(
                 data = load_yaml_string(flow_content)
                 content_hash = hash(flow_content)
             is_eager_flow = cls._is_eager_flow(data)
+            is_async_flow = cls._is_async_flow(kwargs)
             if is_eager_flow:
-                return EagerFlow._load(path=flow_path, data=data, raise_error=raise_error, **kwargs)
+                obj = EagerFlow._load(path=flow_path, data=data, raise_error=raise_error, **kwargs)
```

Comment:
Shall we add a new AsyncEagerFlow class like 
```
class AsyncFlow:
    async def __call__():
       ....
```

Reply:
OKay, of course.